src/sqlancer/MainOptions.java

```diff
@@ -49,6 +49,13 @@
     @Parameter(names = "--password", description = "The password used to log into the DBMS")
     private String password = "sqlancer"; // NOPMD
 
+    @Parameter(names = "--host", description = "The host used to log into the DBMS")
+    private String host = "sqlancer"; // NOPMD
+
+    @Parameter(names = "--port", description = "The port used to log into the DBMS")
+    private String port = "sqlancer"; // NOPMD
```

Comment:
Are these meaningful default values? I assume "sqlancer" is not a valid port.

Reply:
I am wondering whether these should go into `MainOptions` or in the DBMS-specific options. Putting them into `MainOptions` makes sense on a conceptual level. However, putting them into DBMS-specific options (e.g., specifying these in `SQLGlobalState`) might have the advantage of being able to provide default values that are specific to the DBMS. For example, `MySQLGlobalState` could, in its constructor, set the default values for `host` and `port`. These default values would then also be visible in the help options. What do you think?

Reply:
> Are these meaningful default values? I assume "sqlancer" is not a valid port.

It is modified in `xxProvider.java`.

Reply:
I see. Would just leaving them `null` work, given that the `sqlancer` values seem to be placeholders?

Reply:
> I see. Would just leaving them `null` work, given that the `sqlancer` values seem to be placeholders?

I think `sqlancer` is perfect. Because the `host` may be `null`. 
 
```java
if("sqlancer".equals(host)){
     host = default_value;
}
```